types/qrcode-svg/index.d.ts

```diff
@@ -46,20 +44,41 @@ declare class QRCode {
 
 declare namespace QRCode {
     interface Options {
-        /** Default is `"#ffffff"`. */
-        background?: string;
-        /** Default is `"#000000"`. */
-        color?: string;
-        /** The actual content of the QR code */
+        /** QR Code content, the only **required** parameter. */
         content: string;
-        /** Default is `"M"`. */
-        ecl?: "L" | "M" | "H" | "Q";
-        /** Default is `256`. */
-        height?: number;
-        /** Default is `4`. */
+        /** White space padding. `0` for no border. Default is `4`. */
         padding?: number;
-        /** Default is `256`. */
+        /** QR Code width in pixels. Default is `256`. */
         width?: number;
+        /** QR Code height in pixels. Default is `256`. */
+        height?: number;
+        /** Color of modules (squares), color name or hex string. Default is `#000000`. */
+        color?: string;
+        /** Background - color of background, color name or hex string. Default is `#fffff`. */
```

Comment:
Can you remove the `Background` prefix in this comment (to be consistent with the other properties)?